### PR TITLE
Add google-api-python-client as requirement

### PR DESCRIPTION
**Pull request by jimmykane**

### CODE REVIEW

**Comment by tmatsuo** [comment]
Hi @jimmykane 
Thanks for your contributions.
It turns out that we need you to sign our CLA for accepting your contributions. Can you sign one of the following CLAs first?

If you're contributing as an individual person, please sign at:
https://cla.developers.google.com/about/google-individual

and if you're contributing for a company, please sign at:
https://cla.developers.google.com/about/google-corporate

Sorry for not making this point earlier.

**Comment by jimmykane** [comment]
@tmatsuo  Hey hi. I have already done that for NDB, this repo and several more googly stuff. But well I ll do it one more time. :-D

**Comment by tmatsuo** [comment]
@jimmykane Oh, sorry I found your CLA. All clear now.

File: gae-python/appengine_config.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""
+`appengine_config.py` is automatically loaded when Google App Engine
+starts a new instance of your application. This runs before any
+WSGI applications specified in app.yaml are loaded.
+"""
+
+from google.appengine.ext import vendor
+
+# Add any libraries installed in the "vendor" folder.
+vendor.add('vendor')
```
Comment (tmatsuo) [inline gae-python/appengine_config.py]: Can you add a newline?

File: gae-python/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+google-api-python-client
```
Comment (tmatsuo) [inline gae-python/requirements.txt]: Can you add a newline?

**Comment by tmatsuo** [comment]
Can you tweak the README.md for how to install google-api-python-client?

File: gae-python/appengine_config.py (hunk 1 above)
Comment (jimmykane) [inline gae-python/appengine_config.py]: Sure.

**Comment by jimmykane** [comment]
@tmatsuo Should be fine now

**Comment by tmatsuo** [comment]
Do you want to squash the changes before merge? I'm OK with either way.

**Comment by jimmykane** [comment]
Squash looks fine (as they are all related)

**Comment by tmatsuo** [comment]
Ok, so can you squash the changes and push it with -f option? Then I'm going to merge.

**Comment by jimmykane** [comment]
Well I did not know that you could do that with -f and the pull request to remain. Something learned today!

**Comment by tmatsuo** [comment]
Thanks, merging

**Comment by tmatsuo** [comment]
@jimmykane I missed the last sentence. Yup, usually rewriting a history is not a good habit, but I prefer doing this for pull requests.

**Comment by jimmykane** [comment]
Understandable. You are quite precise eh ? Nice. Yes fortunately I did not
have any forks from the repo and was there only for pushing pull requests.
Have a great weekend Takashi!

**Pull request merged**